StrFormat.Test.vbs

```diff
@@ -15,6 +15,8 @@ Import "Tests"
 Import "StrFormat"
 
 Function TC01_StringFormat_WhenHasParameters_ShouldBeReplaced()
+On Error Resume Next
+
     Dim stringFormat, input, result
     
     'Arrange
```

Comment:
Do you want to deal with the error or just want to resume it just right after the error happens.

You could also give it a check on the "Err" object in the end of the execution of your function, if you have error you could ASSERT on your test or you could raise an error for whomever is calling the function.

If Err.Number <> 0 then
     $trace(Err.Number & " " & Err.Description)
     Err.Raise(Err.Number & " " & Err.Description)
End if